site-src/guides/api-overview.md

```diff
@@ -0,0 +1,129 @@
+### AdminNetworkPolicy Actions 
+
+Unlike the NetworkPolicy resource in which each rule represents an allowed
+traffic, AdminNetworkPolicies will enable administrators to set `Pass`,
+`Deny` or `Allow` as the action of each rule. AdminNetworkPolicy rules should
+be read as-is, i.e. there will not be any implicit isolation effects for the Pods
+selected by the AdminNetworkPolicy, as opposed to implicit deny NetworkPolicy rules imply.
+
+- **Pass**: Traffic that matches a `Pass` rule will skip all further rules from all
+  numbered ANPs and instead be enforced by the K8s NetworkPolicies.
+  If there is no K8s NetworkPolicy of BaselineAdminNetworkPolicy rule match
+  match, traffic will be governed by the implementation. For most implementations, 
+  this means "allow", but there may be implementations which have their own policies 
+  outside of the standard Kubernetes APIs.
+- **Deny**: Traffic that matches a `Deny` rule will be dropped.
+- **Allow**: Traffic that matches an `Allow` rule will be allowed.
+
+AdminNetworkPolicy `Deny` rules are useful for administrators to explicitly
+block traffic with malicious in-cluster clients, or workloads that pose security risks.
+Those traffic restrictions can only be lifted once the `Deny` rules are deleted,
+modified by the admin, or overridden by a higher priority rule.
+
+On the other hand, the `Allow` rules can be used to call out traffic in the cluster
+that needs to be allowed for certain components to work as expected (egress to
+CoreDNS for example). Those traffic should not be blocked when developers apply
+NetworkPolicy to their Namespaces which isolates the workloads.
+
+AdminNetworkPolicy `Pass` rules allows an admin to delegate security posture for
+certain traffic to the Namespace owners by overriding any lower precedence Allow
+or Deny rules. For example, intra-tenant traffic management can be delegated to tenant
+admins explicitly with the use of `Pass` rules. More specifically traffic selected 
+by a `Pass` rule will skip any further ANP rule selection, be evaluated against
+any well defined NetworkPolicies, and if not terminated ultimately be evaluated against any 
+BaselineAdminNetworkPolicies. 
+
+### AdminNetworkPolicy Priorities 
+
+Integer priority values were added to the AdminNetworkPolicy API to allow Cluster 
+Admins to express direct and intentional ordering between various ANP Objects.  
+The `Priority` field in the ANP spec is defined as an integer value 
+within the range 0 to 1000 where rules with lower priority values have higher 
+precedence. Regardless of priority value all ANPs have higher precedence than 
+any defined NetworkPolicy or BaselineAdminNetworkPolicy objects.
+
+### AdminNetworkPolicy Rules 
+
+Each ANP should define at least one `Ingress` or `Egress` relevant in-cluster traffic flows. 
+and the Actions that should take place for those flows. In each `gress` rule the user 
+should AT THE MINIMUM define an `Action`, and at least one `AdminNetworkPolicyPeer`.
```

Comment:
I would add a brief description of an AdminNetworkPolicyPeer here, as this is the first time that it is mentioned

Reply:
+1 I want to add a section on "selectors" in a follow up!